src/components/Templates/DefaultSlideTemplate.vue

```diff
@@ -36,14 +36,14 @@
 
       <!-- Carousel Component -->
       <v-flex
-        d-flex
+        align-content-end
         xs12
-        sm8
-        md6>
+        sm12
+        md12
+        lg6>
         <v-carousel
           hide-controls
-          hide-delimiters
-          interval="40000/6">
+          interval="6667"> <!-- Carousel flip time = 40,000 time per slide / 6 images = 6667 (roughly) -->
```

Comment:
Amy, would you be able to make the interval a computed property instead. That way we can have the comment on the method and not in the HTML body.